src/test/java/com/deque/axe/android/AxeTest.java

```diff
@@ -284,10 +287,18 @@ public void jsonTestSpecs() throws IOException {
                         new AxeComparatorInterface() {});
                 comparatorInterface.compare(key.toString(), expected, actual);
               }
+            } else {
+              missingKeys.add(key);
+            }
+          }
+          for (Object missingKey: missingKeys) {
+            if (!ignorePropsList.contains(missingKey.toString())) {
+              unknownMissingKeys.add(missingKey.toString());
             }
           }
         }
 
+        assertEquals(unknownMissingKeys.size(), 0);
```

Comment:
Throw a specific error that describes why this list should be empty.

Reply:
I have added a message to it, please advice if the message does not look good.